app/pb/auth.proto

```diff
@@ -142,19 +142,17 @@ message CompleteTokenLoginReq {
 message AuthReq {
   string user = 1;
   string password = 2;
+
+  // whether to remember the browser or not, see models.UserSession docstring
```

Comment:
Please make the API documentation self-explanatory without refering to implementation internals. (duplicate that information here)

Reply:
OK